src/pf-tooltip/pf-tooltip.component.js

```diff
@@ -0,0 +1,304 @@
+import {default as tmpl} from 'pf-tooltip.template';
+import {pfUtil} from 'pf-utils';
+
+/**
+ * <b>&lt;pf-tooltip&gt;</b> element for Patternfly Web Components
+ *
+ * @example {@lang xml}
+ * <pf-tooltip animation="fade" target="#btn-left" placement="left" delay="100" duration="150" container="#container"></pf-alert>
```

Comment:
Curious why the container must be specified. I understand the default is the body tag; however, couldn't we use the parent of the given target or the pf-tooltip element itself?